tests/test_operation.py

```diff
@@ -6,6 +6,8 @@
 def test_operation(
```

Comment:
can we add a coverage test where we harvest with
`strategy.setEmergencyExit(true)` 

to test the liquidateAllPositions path and get asserts that amounts liquidated are correct?

Reply:
Good call

Reply:
https://github.com/fp-crypto/kashi-strategy/blob/3896b15688319df853c0560d2aa3366faf709da2/tests/test_revoke.py#L30-L33 is there, but maybe I should expand on it ...

Reply:
Done. Let me know if there's more I can do here